doc/README.md

```diff
@@ -34,7 +34,27 @@ $ nix-build doc
 
 If the build succeeds, the manual will be in `./result/share/doc/nixpkgs/manual.html`.
 
-### devmode
+### Development environment
+
+In order to reduce repetition, consider using tools from the provided development environment:
+
+Load it from the Nixpkgs documentation directory with:
```

Comment:
```suggestion
In order to reduce repetition, consider using tools from the provided development environment.
Load it from the Nixpkgs documentation directory with:
```

Double colon? Seems to be an editing artifact

Reply:
Hm, it was intentional as I tend to place them when breaking the flow of prose to show code or instructions. But I see the approach in nix.dev for example to not use them in cases like this so I removed then from here and nixpkgs's doc README.